File: bm-b/src/main/java/com/oayilix/lodestar/demo/moduleb/ThirdActivity.java
package com.oayilix.lodestar.demo.moduleb;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.oayilix.lodestar.annotations.Destination;

@Destination(url = "router://example.com/app/third")
public class ThirdActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_third);
    }
}

File: bm-b/src/main/kotlin/com/oayilix/lodestar/demo/moduleb/ThirdActivity.kt
package com.oayilix.lodestar.demo.moduleb

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.oayilix.lodestar.annotations.Destination

@Destination(url = "router://example.com/app/third")
class ThirdActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_third)
    }
}
